native/c/makefile

```diff
@@ -70,7 +70,7 @@ DLL_CPP_FLAGS=-W "c,lp64,langlvl(extended0x),dll,xplink,exportall"\
 DLL_BND_FLAGS=-W "l,lp64,dll,dynam=dll,xplink,map,list"
 
 CPP_FLAGS=-W "c,lp64,langlvl(extended0x),dll,xplink"\
- -c -D__IBMCPP_TR1__=1
+ -c -D__IBMCPP_TR1__=1 -D_XOPEN_SOURCE=600 -D_XPG4=1 -D__UU
```

Comment:
@traeok Do we still need these flags, since we eliminated use of `pthread`?

Reply:
I don't believe so, now that the mutex is gone we don't need these macros